xc/device.ml

```diff
@@ -1625,7 +1625,7 @@ let vgpu_args_of_nvidia domid vcpus (vgpu:Xenops_interface.Vgpu.nvidia) fds =
 	] in
 	let fd_arg = match fds with
 		| [] -> []
-		| (uuid, _) :: _ -> ["--resumefd"; uuid]
+		| (uuid, _) :: _ -> ["--resumefd=" ^ uuid]
```

Comment:
I don't think that we can do this. The reason that the UUID is used here rather than the actual fd, is that xenopsd hands over its fd to forkexecd, which is going to do the actual forking, and the latter will have to substitute the UUID with its own fd. And for that to work, it needs to be a separate argument.